src/components/TimerModal/TimerBundle.js

```diff
@@ -0,0 +1,76 @@
+import {useContext, useEffect} from "react";
+import {PlayContext, TimerContext} from "./TimerContextProvider";
+import Timer from "./Timer";
+import TimerPlay from "./TimerPlay";
+import style from "./Modal.module.scss";
+
+export default function TimerBundle() {
```

Comment:
Probably don't need to go up to hours.
Better implementation would be to just use seconds and convert that to the display format rather than having a method for each hour, minutes and seconds. There are lots of implementations online which you could have a look at.

Reply:
You're definitely right. Thanks for the idea